bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -70,38 +80,49 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /bitnami/cassandra
+        {{- end }}
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+        {{- end }}
       {{- end }}
       containers:
         - name: cassandra
           command:
+            {{- if .Values.command }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+            {{- else }}
             - bash
             - -ec
-            # Node 0 is the password seeder
             - |
+              # Node 0 is the password seeder
               if [[ $HOSTNAME =~ (.*)-0$ ]]; then
-                echo "Setting node as password seeder"
-                export CASSANDRA_PASSWORD_SEEDER=yes
+                  echo "Setting node as password seeder"
+                  export CASSANDRA_PASSWORD_SEEDER=yes
               else
-                # Only node 0 will execute the startup initdb scripts
-                export CASSANDRA_IGNORE_INITDB_SCRIPTS=1
+                  # Only node 0 will execute the startup initdb scripts
+                  export CASSANDRA_IGNORE_INITDB_SCRIPTS=1
               fi
-              {{ .Values.entrypoint }} {{ .Values.cmd }}
+              /opt/bitnami/scripts/cassandra/entrypoint.sh /opt/bitnami/scripts/cassandra/run.sh
```

Comment:
The entrypoint in the Bitnami container is in the root directory.

Reply:
That changed recently, not it is in that folder